examples/sensors-viewer/src/components/SensorItem.jsx

```diff
@@ -0,0 +1,57 @@
+import {Accordion, Button, Card, ListGroup} from "react-bootstrap";
+import {Images} from "../Images";
+import React from "react";
+
+const _ = require('lodash');
+const moment = require('moment');
+
+export default function SensorItem({availableSensors, sensorValues, sensorSamplingRate}) {
+    return (
+        Object.keys(sensorValues).map((item, index) => {
+            const available = availableSensors[item];
+            const sampling = sensorSamplingRate[item];
+            return <Card key={index} className="main-card border-0 mb-4">
+                <Card.Header className="p-3 bg-white">
+                    <Accordion.Toggle
+                        className="text-dark border-0 font-weight-bold text-uppercase w-100 text-left p-0 text-decoration-none"
+                        as={Button} variant="link" eventKey={index}>
+                        {_.get(available, 'name', item)}
+                        <img src={Images.down_arrow} alt={"down-arrow"}/>
+                    </Accordion.Toggle>
+                </Card.Header>
+                <Accordion.Collapse eventKey={index}>
+                    <Card.Body className="p-3">
+                        <ListGroup className="px-4 py-3 my-2 ">
+                            {available && <ListGroup.Item
```

Comment:
all those `ListGroup.Item` have all the same classes and same internal structure. You can use a function for them
PS: if some of them are not to be rendered, use `.filter` and `.map` functions

Reply:
`ListGroup.Item` not iterate data we are showing different labels and values in each item based on conditions